views/routes/routes.js

```diff
@@ -37,6 +37,7 @@ export const apiRoutes = {
   searchStudies: `${apiPath}/search/studies`,
   subject: `${apiPath}/subjects`,
   shareChart: (chart_id) => `${apiPath}/charts/${chart_id}/share`,
+  updateUserAccess: (uid) => `${apiPath}/admin/${uid}`,
```

Comment:
This endpoint can be used for more than just updating the user access so we should name it something more generic like `updateUser`. It could be used to update any attributes on a user.

```suggestion
  updateUser: (uid) => `${apiPath}/admin/users/${uid}`,
```

Reply:
Ya i thought about that, but update user to me sounds more when a user is updating their preferences. I went with updateUserAccess because an admin is updating a users access to dpdash

Reply:
I think updating user preferences would be a path like `/users/:id/preferences`. These are attributes of a user that are being updated so I think `/admin/users/:id` is the least surprising route.

Reply:
4c885b5249ccbc5570f1e915cc0ac3d7d6147536 👍